lib/Test/Nginx/Util.pm

```diff
@@ -78,6 +81,19 @@ our $ServerAddr = '127.0.0.1';
 
 our $ServerName = 'localhost';
 
+our $Http3SSLCrt = $ENV{TEST_NGINX_HTTP3_CRT};
+our $Http3SSLCrtKey = $ENV{TEST_NGINX_HTTP3_CRT_KEY};
+
+if ($UseHttp2 && $UseHttp3) {
+    die "Ambiguous: both TEST_NGINX_USE_HTTP3 and TEST_NGINX_USE_HTTP2 are set.\n";
+}
+
+if ($UseHttp3 && (!defined $Http3SSLCrt || !defined $Http3SSLCrtKey)) {
+    die "Must set both env TEST_NGINX_HTTP3_CRT and TEST_NGINX_HTTP3_CRT_KEY in global HTTP/3 test mode\n";
+}
+
+our $ServerConfigHttp3 = '';
```

Comment:
Maybe we don't need this variable: setting up tls version and certificate path should be done by the user himself?
cc @agentzh